### ISSUE TITLE
generator-test.js doesn't pass

### ISSUE THREAD

**Issue by arpunk** [vowsjs/kyuri#13]
When I run the generator-test.js script I get the following:

```
When using the Kyuri default code generator, compiling simple.feature
  ✗ should parse correctly
  SyntaxError: Unexpected token ILLEGAL
  at Object.Function (unknown source)
  at Function.template (/Users/ricardolanziano/.local/lib/node/.npm/underscore/1.1.2/package/underscore.js:656:16)
  at Object.renderFeature (/Users/ricardolanziano/kyuri/lib/kyuri/generator.js:73:18)
  at Object.compile (/Users/ricardolanziano/kyuri/lib/kyuri/generator.js:42:26)
  at Object.compile (/Users/ricardolanziano/kyuri/lib/kyuri/core.js:27:22)
  at Object.<anonymous> (/Users/ricardolanziano/kyuri/test/generator-test.js:44:15)
  at runTest (/Users/ricardolanziano/.local/lib/node/.npm/vows/0.5.2/package/lib/vows.js:95:26)
  at EventEmitter.<anonymous> (/Users/ricardolanziano/.local/lib/node/.npm/vows/0.5.2/package/lib/vows.js:73:9)
  at EventEmitter.emit (events:41:20)
  at /Users/ricardolanziano/.local/lib/node/.npm/vows/0.5.2/package/lib/vows/context.js:29:52

When using the Kyuri default code generator, compiling complex.feature
  ✗ should parse correctly
  SyntaxError: Unexpected token ILLEGAL
  at Object.Function (unknown source)
  at Function.template (/Users/ricardolanziano/.local/lib/node/.npm/underscore/1.1.2/package/underscore.js:656:16)
  at Object.renderFeature (/Users/ricardolanziano/kyuri/lib/kyuri/generator.js:73:18)
  at Object.compile (/Users/ricardolanziano/kyuri/lib/kyuri/generator.js:42:26)
  at Object.compile (/Users/ricardolanziano/kyuri/lib/kyuri/core.js:27:22)
  at Object.<anonymous> (/Users/ricardolanziano/kyuri/test/generator-test.js:51:15)
  at runTest (/Users/ricardolanziano/.local/lib/node/.npm/vows/0.5.2/package/lib/vows.js:95:26)
  at EventEmitter.<anonymous> (/Users/ricardolanziano/.local/lib/node/.npm/vows/0.5.2/package/lib/vows.js:73:9)
  at EventEmitter.emit (events:41:20)
  at /Users/ricardolanziano/.local/lib/node/.npm/vows/0.5.2/package/lib/vows/context.js:29:52
```

The other tests run just fine. Im using node 0.2.4, vows 0.5.2, underscore 1.1.2 and kyuri from github on Mac OS X (snow leopard)

If you need any further info, please let me know.

**Comment by indexzero** [comment]
I've updated all the tests and merged in the 0.2.0 branch (although it is currently incomplete). Let me know if you run into anymore issues

**Comment by arpunk** [comment]
step-generator-test.js and generator-test.js both fail with the following backtrace:

```
 SyntaxError: Unexpected string
  at Object.Function (unknown source)
  at Function.template (/Users/ricardolanziano/.local/lib/node/.npm/underscore/1.1.2/package/underscore.js:656:16)
  at [object Object].renderFeature (/Users/ricardolanziano/kyuri/lib/kyuri/generator.js:81:18)
  at [object Object].compile (/Users/ricardolanziano/kyuri/lib/kyuri/generator.js:58:26)
  at Object.compile (/Users/ricardolanziano/kyuri/lib/kyuri/core.js:31:40)
  at Object.<anonymous> (/Users/ricardolanziano/kyuri/test/step-generator-test.js:46:15)
  at runTest (/Users/ricardolanziano/.local/lib/node/.npm/vows/0.5.2/package/lib/vows.js:95:26)
  at EventEmitter.<anonymous> (/Users/ricardolanziano/.local/lib/node/.npm/vows/0.5.2/package/lib/vows.js:73:9)
  at EventEmitter.emit (events:41:20)
  at /Users/ricardolanziano/.local/lib/node/.npm/vows/0.5.2/package/lib/vows/context.js:29:52
```

The other tests pass just fine.

**Comment by indexzero** [comment]
Ok, this seems like a regression between underscore 1.1.0 and 1.1.2. Can you confirm for me? Try this:

<pre>
  npm uninstall underscore
  npm install underscore@1.1.0
</pre>


Then re-run the tests. I will try to follow up with this later, but hopefully that should be enough to get you rolling. Any commits to help track down this error are welcome :)

**Comment by indexzero** [comment]
Fixed in `kyuri@0.2.0`
